src/components/timecard/edit-shift-timecard/EditShiftTimecard.jsx

```diff
@@ -35,6 +35,14 @@ const EditShiftTimecard = ({ timeEntry, timeEntriesIndex }) => {
     });
   };
 
+  // const isFinishTimeNextDay = (startTimeValue, finishTimeValue) => {
```

Comment:
this function name suggests it's returning a boolean when in fact its giving us the output text.
Could we either change the function name, or move the display text into the jsx.